### PR TITLE
adding aspects to catch basins - also a couple of missing design attr…

### PR DESCRIPTION
**Pull request by BenTWhite**
adding  new aspects to the catch basins for apply the inlet capture method...

- added a couple new attributes to the design settings

### CODE REVIEW

File: Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml [hunk 1]
```diff
@@ -142,9 +148,34 @@
...
+      <Class class="CatchBasin"/>
+    </Source>
+    <Target multiplicity="(1..1)" roleLabel="owned by" polymorphic="true">
+      <Class class="InletCaptureMethodAspect"/>
+    </Target>
+  </ECRelationshipClass>
+
+  <ECEntityClass typeName="PercentCapture" modifier="Sealed" displayLabel="Percent Capture">
+    <BaseClass>InletCaptureMethodAspect</BaseClass>
+    <ECProperty propertyName="CapturePercent" typeName="double" displayLabel="Capture Percent" description="Set the percent of incoming inflow to be captured by the inlet." category="HydraulicData"/>
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml:172]: You could assign a kindOfQuantity to this unitized property. I notice neither the RoadRailUnits nor the AecUnits schemas have a KOQ for the involved "phenomenon" (PERCENTAGE). Thus, you could add one that you can run with - probably to RoadRailUnits. e.g.

```
<KindOfQuantity typeName="PERCENTAGE" displayLabel="Road &amp; Rail Percentage" persistenceUnit="u:PERCENT" presentationUnits="f:DefaultRealU(2)[u:PERCENT];f:DefaultRealU(4)[u:DECIMAL_PERCENT]" relativeError="0.0001"/>
```

File: Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml [hunk 2]
```diff
@@ -336,5 +367,8 @@
     <ECProperty propertyName="LimitSectionSize" typeName="boolean" displayLabel="Limit Section Size" category="HydraulicData"/>
     <ECProperty propertyName="MaximumRise" typeName="double" displayLabel="Rise (Maximum)" category="HydraulicData"/>
     <ECProperty propertyName="MeasureCoverTo" typeName="MeasureCoverTo" displayLabel="Measure Cover To" category="HydraulicData"/>
-  </ECEntityClass>
+		<ECProperty propertyName="IsPartFullDesign" typeName"boolean", displayLabel="Is Part Full Design" category="HydraulicData"/>
+		<ECProperty propertyName="ConduitPercentageFull" typeName="double" displayLabel="Percentage Full" category="HydraulicData"/>
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml:371]: Same comment about a massing "PERCENTAGE" KOQ.

Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml]: same as above

File: Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml (hunk 1 above)
Comment (BenTWhite) [reply Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml]: added the PERCENTAGE KindOfQuantity to the two fields....

also I was getting this error:

_***** Schema RoadRailUnits Validation Results *****
Schema file path: D:\learning\bis-schemas\Domains\2-DisciplinePhysical\Civil\RoadRailUnits\RoadRailUnits.ecschema.xml
"##vso[task.logissue type=error]Error BIS-1001: KindOfQuantity 'RoadRailUnits.PERCENTAGE' has persistence unit of unit system 'Units.INTERNATIONAL' but must have an SI unit system"_

so I updated the Units.ecschema.xml file to get passed the error

File: System/Units.ecschema.xml [hunk 3]
```diff
@@ -479,7 +479,7 @@
     <Unit typeName="IN_HG_AT_60F" phenomenon="PRESSURE" unitSystem="USCUSTOM" definition="PA" numerator="3.37685e3" displayLabel="inHg@60°F" description="See http://physics.nist.gov/cuu/pdf/sp811.pdf Appendix B." />
     <Unit typeName="PA_PER_M" phenomenon="PRESSURE_GRADIENT" unitSystem="SI" definition="PA*M(-1)" displayLabel="Pa/m" />
     <Unit typeName="BAR_PER_KM" phenomenon="PRESSURE_GRADIENT" unitSystem="METRIC" definition="BAR*KM(-1)" displayLabel="bar/km" />
-    <Unit typeName="PERCENT" phenomenon="PERCENTAGE" unitSystem="INTERNATIONAL" definition="ONE" displayLabel="%" description="Unit used to show a ratio as a percentage in the UI."/>
+    <Unit typeName="PERCENT" phenomenon="PERCENTAGE" unitSystem="SI" definition="ONE" displayLabel="%" description="Unit used to show a ratio as a percentage in the UI."/>
```
Comment (diegoalexdiaz) [inline System/Units.ecschema.xml:482]: @ColinKerr, Ben had to do this in order to pass schema Validation. What's your take on this one?

Comment (ColinKerr) [reply System/Units.ecschema.xml:482]: The persistence unit we decided on is DECIMAL_PERCENT.  so 100% is 1.  With this unit 100% would be 100.

There was push back on listing DECIMAL_PERCENT as SI so we make it INTERNATIONAL and added a rule that allowed it: https://github.com/iTwin/bis-schema-validation/blob/c77f3a4d8247e19f4bd98870b80a0637157bc6f4/bis-rules/src/BisRules.ts#L1045

I suppose this should be better documented or we should change DECIMAL_PERCENT to SI

Comment (BenTWhite) [reply System/Units.ecschema.xml]: I reverted the change to the unitsschema and switch to the Decimal_Percent unit for the kind of quantity in the railroad units

**Review by diegoalexdiaz — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Pull request merged**